javascript/javascript1/week2/Weather wear.js

```diff
@@ -0,0 +1,17 @@
+function yourClothToWear(temperature){
+if (temperature < 0) 
+{return "Scarf, Coat and Gloves" ;
+ }
+ else if (temperature > 0 && temperature < 10) {
+return "Rain Coat";
+  }
+  else if (temperature < 21) {
+    return "shirt and jeans";
+  } 
+  else {
+    return "shorts and t-shirt";
+  }
+    }
+  
+const clothToWear=yourClothToWear(15)
+  console.log("wear"+clothToWear);
```

Comment:
Really nice solution. Well done! 😊

**A few observations and reflections** 
- for the if else statements, the chronology matters. What would happen if you only wrote else if(temperature < 10), in your first if else statement? Why? Try it out 😊 
- in the finale console.log you need a space between "wear" and your const clothToWear. 
- really good application of const here 👌